darwin/importer/formats/superannotate.py

```diff
@@ -208,37 +229,83 @@ def _to_cuboid_annotation(cuboid: Dict[str, Any], classes: List[Dict[str, Any]])
+    name: str = str(instance_class.get("name"))
     points: List[Point] = _map_to_list(_tuple_to_point, _group_to_list(data, 2, 0))
 
-    return make_polygon(f"{name}-polygon", points)
+    attributes: Optional[SubAnnotation] = _get_attributes(polygon, instance_class)
+    subannotations: Optional[List[SubAnnotation]] = None
+    if attributes:
+        subannotations = [attributes]
+    return make_polygon(f"{name}-polygon", points, None, subannotations)
 
 
 def _to_line_annotation(line: Dict[str, Any], classes: List[Dict[str, Any]]) -> Annotation:
     data: List[float] = cast(List[float], line.get("points"))
     class_id: int = cast(int, line.get("classId"))
-    name: str = _find_class_name(class_id, classes)
+    instance_class: Dict[str, Any] = _find_class(class_id, classes)
+    name: str = str(instance_class.get("name"))
     points: List[Point] = _map_to_list(_tuple_to_point, _group_to_list(data, 2, 0))
+    attributes: Optional[SubAnnotation] = _get_attributes(line, instance_class)
+    subannotations: Optional[List[SubAnnotation]] = None
+    if attributes:
+        subannotations = [attributes]
 
-    return make_line(f"{name}-polyline", points)
+    return make_line(f"{name}-polyline", points, subannotations)
 
 
-def _find_class_name(class_id: int, classes: List[Dict[str, Any]]) -> str:
+def _find_class(class_id: int, classes: List[Dict[str, Any]]) -> Dict[str, Any]:
     obj: Optional[Dict[str, Any]] = next((class_obj for class_obj in classes if class_obj.get("id") == class_id), None)
 
     if obj is None:
         raise ValueError(
             f"No class with id '{class_id}' was found in {classes}.\nCannot continue import, pleaase check your 'classes.json' file."
         )
 
-    return str(obj.get("name"))
+    return obj
+
+
+def _get_attributes(instance: Dict[str, Any], instance_class: Dict[str, Any]) -> Optional[SubAnnotation]:
+    attribute_info: List[Dict[str, int]] = cast(List[Dict[str, int]], instance.get("attributes"))
+    groups: List[Dict[str, Any]] = cast(List[Dict[str, Any]], instance_class.get("attribute_groups"))
+    all_attributes: List[str] = []
+
+    for info in attribute_info:
+        info_group_id: int = cast(int, info.get("groupId"))
+        attribute_id: int = cast(int, info.get("id"))
+
+        for group in groups:
+            group_id: int = cast(int, group.get("id"))
+
+            if info_group_id == group_id:
+                group_attributes: List[AttributeGroup] = cast(List[AttributeGroup], group.get("attributes"))
+                attribute: Optional[AttributeGroup] = next(
+                    (attribute for attribute in group_attributes if attribute.get("id") == attribute_id), None
+                )
+
+                if attribute is None:
+                    raise ValueError(f"No attribute data found for {info}.")
+
+                final_attribute: str = f"{str(group.get('name'))}:{str(attribute.get('name'))}"
+                all_attributes.append(final_attribute)
```

Comment:
My linter would complain we're over-nesting here, so I'd propose this instead
```suggestion
            if info_group_id != group_id:
                continue
                
            group_attributes: List[AttributeGroup] = cast(List[AttributeGroup], group.get("attributes"))
            attribute: Optional[AttributeGroup] = next(
                (attribute for attribute in group_attributes if attribute.get("id") == attribute_id), None
            )

            if attribute is None:
                raise ValueError(f"No attribute data found for {info}.")

            final_attribute: str = f"{str(group.get('name'))}:{str(attribute.get('name'))}"
            all_attributes.append(final_attribute)
```

Reply:
Your linter is likely right. We should probably use the same linters as I am surprised mine didn't complain. 
Anyway, suggested committed !